QUICKSTART.md

```diff
@@ -0,0 +1,10 @@
+On this quickstart guide, we'll deploy the application in docker
+
+# PREREQUESITES
+- Docker 
+- (For the complete list, check the Readme and the dockerfile)
+
+1. Run the `install_docker_image.sh`, it will build the docker image and create the configuration files
```

Comment:
Missing

- Key generation (Pub/Priv)
- report_details_example.yaml modification
- get the packet using extractor (using notsend)
- Api key generation on the DSM
- basic run example `as one would've done it`

Reply:
Key generation is not strictly necessary for connecting to the DSM and generating reports, only if the extractor is involved. As the usage of the extractor requires more "moving parts", In my opinion it is not required of the "quickstart"

Will add API key generation and config details

Reply:
it is for basic usage of the docker, it is not for using with our cloud implementation

Reply:
if a user wants to test by it self, (and test their modifications in the code) they have to know how to make it.

Reply:
The complete README.md details usage of the key under "KEY GENERATION AND USAGE". 

A quickstart should be short and concise, not delving into advanced options and simply providing the user with a step-by-step of what to do to achieve _quick_ results. Given this, I suggest leaving documentation of the keys under the README.md

Reply:
indeed it should be short and concise, but it should mainly also be complete and funcional,

you **can't** develop or even extract without a key
making keys are basic Knowledge for using the extractor and test containers,

the public key given by the project is only to be used with our production extractor w/ our cloud.

the quickstart should be without sugar what is need to start to end to use the extractor and the container in a single file.

Reply:
f5

Reply:
the better you explain the less you have to repeat yourself. - me rn

Reply:
The quickstart so far hasn't even mentioned the extractor tool, nor how to generate an extractor binary. What then is the purpose of explaining key generation here?

Reply:
because you can't basically do nothing without it?

A key is needed to parse with the extractor and it is also needed to use the docker container in dev and production cases

the basic of this file is
follow what is written here and the project will work.
if you follow what you added on these steps, it will **not** work. the project requires a key pair.

there`s absolutely no case where you would not need a key pair.